Tests/Tracking/Util/DeviceAttributesProviderTest.swift

```diff
@@ -55,23 +56,6 @@ class DeviceAttributesProviderTest: UnitTest {
 
         waitForExpectations()
     }
-
-    func test_getSdkVersionAttribute_givenNotUsingSdkWrapper_expectGetSDKVersion() {
```

Comment:
Shouldn't we still have test to verify correct SDK version being attached? Or do we plan add more tests for `DeviceAttributesProvider` in later PR?

Reply:
I would add the test cases in another PR. This PR is just a fix to the broken test cases that was being a blocker to run other tests cases.